sklearn/linear_model/ransac.py

```diff
@@ -188,6 +188,10 @@ def fit(self, X, y):
         y : array-like, shape = [n_samples] or [n_samples, n_targets]
             Target values.
 
+        sample_weight: numpy array of shape [n_samples]
```

Comment:
`numpy array` -> `array-like`

Also indent the next line just 4 spaces to the first line.
Look at how all the other parameters are documented